### PR TITLE
fix: logging consistency

### PR DESCRIPTION
**Pull request by aarnphm**
- refactor(cli): fixing types and restore previous state for environment set
- fix: logging level consistency

This PR:
Screenshot 2022-07-08 at 22 21 04

on `main`:
Screenshot 2022-07-08 at 22 22 00

Notice the last two lines from this PR that `main` is missing. Those are bentoml logs.

### CODE REVIEW

File: bentoml/_internal/cli/click_utils.py [hunk 1]
```diff
@@ -104,36 +112,31 @@ def wrapper(
...
+                if (
+                    cmd_group.name in cli_events_map
+                    and command_name in cli_events_map[cmd_group.name]
+                ):
+                    return cli_events_map[cmd_group.name][command_name](
                         cmd_group=cmd_group.name,
                         cmd_name=command_name,
+                        return_value=return_value,
                     )
+                return CliEvent(cmd_group=cmd_group.name, cmd_name=command_name)
```
Comment (aarnphm) [inline bentoml/_internal/cli/click_utils.py]: To fix pyright type error.

File: bentoml/_internal/cli/click_utils.py [hunk 2]
```diff
@@ -32,37 +37,39 @@ class ClickFunctionWrapper(t.Protocol[P]):
         def __call__(*args: P.args, **kwargs: P.kwargs) -> t.Callable[P, t.Any]:
             ...
 
-    WrappedCLI = t.Callable[P, ClickFunctionWrapper[t.Any]]
+    WrappedCLI = t.Callable[P, ClickFunctionWrapper[P]]
```
Comment (aarnphm) [inline bentoml/_internal/cli/click_utils.py]: fix some type hint.

File: bentoml/_internal/log.py [hunk 3]
```diff
@@ -88,7 +88,7 @@ def configure_logging():
         CLI_LOGGING_CONFIG["root"]["level"] = logging.DEBUG
     else:
         CLI_LOGGING_CONFIG["loggers"]["bentoml"]["level"] = logging.INFO
-        CLI_LOGGING_CONFIG["root"]["level"] = logging.WARNING
+        CLI_LOGGING_CONFIG["root"]["level"] = logging.INFO
```
Comment (aarnphm) [inline bentoml/_internal/log.py:91]: This should be INFO so that it doesn't quiet server script logs

File: bentoml/_internal/log.py [hunk 4]
```diff
@@ -115,7 +115,7 @@ def configure_server_logging():
         SERVER_LOGGING_CONFIG["root"]["level"] = logging.DEBUG
     else:
         SERVER_LOGGING_CONFIG["loggers"]["bentoml"]["level"] = logging.INFO
-        SERVER_LOGGING_CONFIG["root"]["level"] = logging.WARNING
+        SERVER_LOGGING_CONFIG["root"]["level"] = logging.INFO
```
Comment (aarnphm) [inline bentoml/_internal/log.py:118]: same here

**Review by aarnphm — COMMENTED** [review]
cc @sauyon @ssheng

File: bentoml/_internal/log.py (hunk 3 above)
Comment (sauyon) [reply bentoml/_internal/log.py:91]: Why aren't the server script logs logging from the bentoml namespace?

Comment (sauyon) [reply bentoml/_internal/log.py:91]: We should configure those logs manually; we don't want to allow info logs from any other libraries through by default, I think.

**Comment by ssheng** [comment]
Seems like circus is configuring logs (notice the format difference) when it shouldn't. Is there an option for skipping the circus log configuration, similar to skipping log configuration in Starlette.

**Comment by aarnphm** [comment]
i believe circus is fine to show in this case, since it is a development mode. imo the config should be at least consistent. We can decide what to show or what not to.

**Pull request closed without merging**